pkg/controllers/nodeclaim/lifecycle/registration_test.go

```diff
@@ -55,6 +59,7 @@ var _ = Describe("Registration", func() {
 				})
 			}
 			nodeClaim := test.NodeClaim(nodeClaimOpts...)
+			nodePool.StatusConditions().SetUnknown(v1.ConditionTypeNodeRegistrationHealthy)
```

Comment:
Do you need to set this to Unknown? Can't the lack of its existence also serve as it being treated as Unknown?